bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -155,9 +156,18 @@ class App extends Component {
 
   componentDidUpdate(prevProps) {
     const {
-      meetingMuted, notify, currentUserEmoji, intl, hasPublishedPoll,
+      meetingMuted,
+      notify,
+      currentUserEmoji,
+      intl,
+      hasPublishedPoll,
+      randomlySelectedUser,
+      currentUserId,
+      mountModal,
     } = this.props;
 
+    if (randomlySelectedUser === currentUserId) mountModal(<RandomUserSelectContainer />);
```

Comment:
the modal appears again and again when the selected user resizes the window. Should randomlySelectedUser be cleared when the moderator closes the modal?